pkgs/build-support/ocaml/oasis.nix

```diff
@@ -0,0 +1,31 @@
+{ buildOcaml, ocaml_oasis }:
+
+{ name, version, buildInputs ? [], ...
```

Comment:
The variables `name` and `version` are bound but never used. Is it on purpose?

Reply:
You're right. lets change it to:

```suggestion
{buildInputs ? [], ...
```